apps/wing-console/console/server/src/router/app.ts

```diff
@@ -527,6 +525,95 @@ export const createAppRouter = () => {
         ) as IFunctionClient;
         await client.invoke("");
       }),
+
+    "app.invokeFileBrowserGetQuery": createProcedure
```

Comment:
Maybe we could use different prefix and files to implement this (also, why `...GetQuery` instead of `get`?). For example, `router/file-browser.ts` and `fileBrowser.getObject`?

Reply:
I agree, we should have a separate router file for each custom resource

Reply:
done!